docs/bokeh/source/docs/user_guide/interaction/linking.rst

```diff
@@ -33,11 +37,17 @@ selection extends to glyph renderers that plot only a subset of data from a data
 source, see :ref:`ug_basic_data_linked_selection_with_filtering`.
 
 The following code shows an example of linked brushing between circle glyphs on
-two different |figure| calls.
+two different |figure| calls:
 
 .. bokeh-plot:: __REPO__/examples/interaction/linking/linked_brushing.py
     :source-position: above
 
+A more sophisticated example below demonstrates linked selection between a
+``DataTable`` widget and a scatter plot:
+
+.. bokeh-plot:: __REPO__/examples/interaction/linking/data_table_plot.py
+    :source-position: above
+
```

Comment:
@tcmetzger adding this example here anticipates the "widget" chapter, but I think its an interesting and important example (that has been hidden buried in `examples/models` for too long) and that adding it here helps make this chapter less meager. Do we need to add a sentence with a forward link to "widgets"?